plugins/in_head/in_head.c

```diff
@@ -434,6 +382,52 @@ static int in_head_exit(void *data, struct flb_config *config)
     return 0;
 }
 
+/* Configuration properties map */
+static struct flb_config_map config_map[] = {
+    // lines
+    {
+     FLB_CONFIG_MAP_STR, "file", "",
+     0, FLB_TRUE, offsetof(struct flb_in_head_config, filepath),
```

Comment:
I think the data type of `filepath` should be `flb_sds_t` since config_map framework allocates as flb_sds_t.
https://github.com/fluent/fluent-bit/blob/v1.8.12/src/flb_config_map.c#L685